addons/refuel/XEH_postInit.sqf

```diff
@@ -1,3 +1,3 @@
 #include "script_component.hpp"
 
-["medical_onUnconscious", {_this call FUNC(handleOnUnconscious)}] call EFUNC(common,addEventHandler);
+["medical_onUnconscious", {_this call FUNC(handleUnconscious)}] call EFUNC(common,addEventHandler);
```

Comment:
Same error in `rearm` according to the RPT.
https://github.com/acemod/ACE3/blob/master/addons/rearm/XEH_postInit.sqf#L3